homeassistant/components/tuya/switch.py

```diff
@@ -513,6 +513,34 @@
             icon="mdi:account-lock",
             entity_category=EntityCategory.CONFIG,
         ),
+        SwitchEntityDescription(
```

Comment:
Added `fan_cool` switch to Fan (fs) with name `Cool Wind Natural` for evaluation after comment in previous PR.
https://github.com/home-assistant/core/pull/64947#discussion_r792974132
![image](https://user-images.githubusercontent.com/31328123/151274812-8b661d7a-4803-4a9c-9a94-678181d71c78.png)